cidr-filtering-cni

```diff
@@ -0,0 +1,252 @@
+#!/bin/bash
+
+set -e
+set -o pipefail
+
+stdin=$(cat /dev/stdin)
+logFile="/var/log/cidr-filtering-cni.log"
+
+exec 3>&1
+exec 2>> $logFile
+
+NFT_TABLE_NETDEV=netdev
+NFT_TABLE_BRIDGE=bridge
+NFT_TABLE=filter
+
+NFT_INGRESS_CHAIN=ingress
+NFT_INGRESS_HOOK=ingress
+
+NFT_POSTROUTING_CHAIN=postrouting
+NFT_EGRESS_HOOK=postrouting
+
+get_object() {
+    local json_object="$1"
+    local json_path="$2"
+    echo "$json_object" | jq -c "$json_path"
+}
+
+get_array_items() {
+    local json_object="$1"
+    local json_path="$2"
+    echo "$json_object" | jq -c $json_path | jq -c ".[]"
+}
+
+get_array_len() {
+    local json_object="$1"
+    local json_path="$2"
+    echo "$json_object" | jq -c "$json_path" | jq ". | length"
+}
+
+for_json_array() {
+    local json_object="$1"
+    local json_path="$2"
+    local f="$3"
+    set -- "${@:4}"
+    for item in $(get_array_items "$json_object" "$json_path"); do
+        $f $item $@
+    done
```

Comment:
but there aren't actually usages of more than 3 params on `for_json_array` calls right ?
so maybe we dont need this extra support for $@ ?
(it is nice, but make it little heavier)

Reply:
Indeed, it's not used. Removed